docs/source/ja/_toctree.yml

```
@@ -337,7 +349,14 @@
         title: CLIPSeg
       - local: model_doc/clvp
         title: CLVP
+      - local: model_doc/data2vec
+        title: Data2Vec
       title: マルチモーダルモデル
+    - isExpanded: false
+      sections:
+      - local: model_doc/decision_transformer
+        title: 意思決定トランスフォーマー
```

No need to translate the model name since it's a proper noun

```suggestion
        title: Decision Transformer
```